### ISSUE TITLE
Please make it possible to install multiple community operators using helm in different namespaces

### ISSUE THREAD

**Issue by denis-zhdanov** [mongodb/mongodb-kubernetes-operator#1282]
**What did you do to encounter the bug?**

1. Prepare local k8s environment:
    ```
    minikube delete
    minikube start
    ```
2. Prepare different namespaces:
    ```
    kubectl create namespace n1
    namespace/n1 created
    
    kubectl create namespace n2
    namespace/n2 created
    
    kubectl get namespace
    NAME              STATUS   AGE
    default           Active   2m34s
    kube-node-lease   Active   2m35s
    kube-public       Active   2m35s
    kube-system       Active   2m35s
    n1                Active   73s
    n2                Active   7s
    ```
3. Try to install mongo community operatos in those namespaces following [documentation link](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md#install-in-a-different-namespace-using-helm)
    ```
    helm install community-operator mongodb/community-operator --namespace n1
    NAME: community-operator
    LAST DEPLOYED: Thu May  4 15:24:28 2023
    NAMESPACE: n1
    STATUS: deployed
    REVISION: 1
    TEST SUITE: None
    
    helm install community-operator mongodb/community-operator --namespace n2
    Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "mongodbcommunity.mongodbcommunity.mongodb.com" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "n2": current value is "n1"
    ```

**What did you expect?**
It was expected to have different mongo community operators installed and operating in different namespaces (CI environment when many builds run in parallel)

**What happened instead?**
Attempt to install any subsequent mongo community operator in a different namespaces fails

**Operator Information**
the latest as of May 4, 2023
```
kubectl describe deployment.apps/mongodb-kubernetes-operator --namespace n1
Name:                   mongodb-kubernetes-operator
Namespace:              n1
CreationTimestamp:      Thu, 04 May 2023 15:24:28 +0800
Labels:                 app.kubernetes.io/managed-by=Helm
                        owner=mongodb
Annotations:            deployment.kubernetes.io/revision: 1
                        email: support@mongodb.com
                        meta.helm.sh/release-name: community-operator
                        meta.helm.sh/release-namespace: n1
Selector:               name=mongodb-kubernetes-operator
Replicas:               1 desired | 1 updated | 1 total | 1 available | 0 unavailable
StrategyType:           RollingUpdate
MinReadySeconds:        0
RollingUpdateStrategy:  1 max unavailable, 25% max surge
Pod Template:
  Labels:           name=mongodb-kubernetes-operator
  Service Account:  mongodb-kubernetes-operator
  Containers:
   mongodb-kubernetes-operator:
    Image:      quay.io/mongodb/mongodb-kubernetes-operator:0.7.9
    Port:       <none>
    Host Port:  <none>
    Command:
      /usr/local/bin/entrypoint
    Limits:
      cpu:     1100m
      memory:  1Gi
    Requests:
      cpu:     500m
      memory:  200Mi
    Environment:
      WATCH_NAMESPACE:              (v1:metadata.namespace)
      POD_NAME:                     (v1:metadata.name)
      OPERATOR_NAME:               mongodb-kubernetes-operator
      AGENT_IMAGE:                 quay.io/mongodb/mongodb-agent:12.0.15.7646-1
      VERSION_UPGRADE_HOOK_IMAGE:  quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook:1.0.6
      READINESS_PROBE_IMAGE:       quay.io/mongodb/mongodb-kubernetes-readinessprobe:1.0.13
      MONGODB_IMAGE:               mongo
      MONGODB_REPO_URL:            docker.io
    Mounts:                        <none>
  Volumes:                         <none>
Conditions:
  Type           Status  Reason
  ----           ------  ------
  Available      True    MinimumReplicasAvailable
  Progressing    True    NewReplicaSetAvailable
OldReplicaSets:  <none>
NewReplicaSet:   mongodb-kubernetes-operator-7779c8f68f (1/1 replicas created)
Events:
  Type    Reason             Age    From                   Message
  ----    ------             ----   ----                   -------
  Normal  ScalingReplicaSet  2m41s  deployment-controller  Scaled up replica set mongodb-kubernetes-operator-7779c8f68f to 1
```

**Kubernetes Cluster Information**
```
minikube version
minikube version: v1.28.0
commit: 986b1ebd987211ed16f8cc10aed7d2c42fc8392f
```

**Additional context**
M1 macbook

**Comment by Jamison1** [comment]
This is possible.  The error seems to be related to trying to Helm install community-operator the 2nd time and the CRD's were automatically installed with the first Helm install.

There are several ways to handle this.

I used [Helm to install the CRD's](https://github.com/mongodb/helm-charts/tree/main/charts/community-operator-crds) and then for each Helm install of the community-operator I set the following:

```
values:
  community-operator-crds:
    enabled: false
```
Also for the `MongoDBCommunity` & `Secret` files or if you're using the link to the `mongodb.com_v1_mongodbcommunity_cr.yaml` I created separate files and changed multiple items to insure uniqueness between the different installations.
